app/forms/claimants_detail.rb

```diff
@@ -31,12 +31,12 @@ def to_h # rubocop:disable Metrics/MethodLength
   end
 
   validates :claimants_name,
-    persons_name: true,
-    allow_blank: true
+            persons_name: true,
+            allow_blank: true
   validates :agree_with_employment_dates, inclusion: { in: [true, false] }, allow_blank: true
   validates :employment_start, :employment_end,
-    date: true, allow_blank: true,
-    if: :disagree_with_employment_dates?
+            date: true, allow_blank: true,
```

Comment:
@garytaylor I think the `date: true` here isn't catching EtDateType::InvalidDate - or, accepting it as a Date type. I could perhaps add the EtDateType::InvalidDate check here?